app/controllers/application_controller.rb

```diff
@@ -1,3 +1,3 @@
 class ApplicationController < ActionController::Base
-  protect_from_forgery with: :exception
```

Comment:
why did you remove this one?

Reply:
Sorry Maam was debugging, will add it again